docs/grpc/docs.md

```
@@ -3047,7 +3047,15 @@ Additionally, the first and last points of each GeoLineString must be the same.
 
 
 
-<a name="qdrant-HardwareUsage"></a>
+<a name="qdrant-Usage"></a>
+
+### Usage
+
+
+| Field                  | Type                     | Label | Description |
+|------------------------|--------------------------| ----- | ----------- |
+| hardware                 | [HardwareUsage](#qdrant-HardwareUsage) |  |  |
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)